.travis.yml

```diff
@@ -219,7 +219,14 @@ jobs:
         - bash ./tests/travis/install-sqlsrv-dependencies.sh
         - bash ./tests/travis/install-mssql-pdo_sqlsrv.sh
         - bash ./tests/travis/install-mssql.sh
-
+    - stage: Test
```

Comment:
What if we reuse an existing build instead of introducing a new one? It will save us build time.